ietf/meeting/tests_views.py

```diff
@@ -294,6 +294,8 @@ def test_meeting_agenda(self):
             (slot.time + slot.duration).astimezone(meeting.tz()).strftime("%H%M"),
         ))
         self.assertContains(r, f"shown in the {meeting.tz()} time zone")
+        updated = meeting.updated().astimezone(meeting.tz()).strftime("%Y-%m-%d %H:%M:%S %Z")
```

Comment:
Please double check for me that this isn't prone to race condition. Same with line 314 below. Otherwise, I'm happy to `patch` it with a mock return value instead of directly calling `meeting.updated()`.

P.S. I encountered some errors locally earlier, but I haven't been able to reproduce them since, which made me think that it might be related to race condition.

Reply:
the tests all run in a single thread in a single process for now - you can't be fighting a race.

It would be _nice_ to be able to use threading, but we have dependencies that would have to change (and we would have to rewrite our custom harness around the django test framework.)

Reply:
Thanks. Maybe it was a bug I accidentally introduced while updating the code locally. It doesn't appear to be there anymore.

Faster tests would be nice in the future.